static/datachecks/js/datacheck_jobs.js

```diff
@@ -0,0 +1,218 @@
+function expandIcon(value, row , index){
+  return '<span title="Click for more info" style="cursor: pointer; color:blue" class="fas fa-plus" onclick="row_details('+ index +')">+</span> <span class="fa fa-plus"></span>'
+}
+
+function row_details(id){
+ console.log(id);
+ let table = $('#table');
+ console.log(table);
+ table.bootstrapTable('expandRow', id);
+}
+
+function downloadFormat(value, row){
+
+  let cursor_download = 'not-allowed';
+  let download_event = 'pointer-events:none';
+  if(row.status == 'complete'){
+    cursor_download = 'pointer';
+    download_event = ''; 
+  }
+  return '<a title="Dowload DC.. " style="cursor:'+ cursor_download +'; '+ download_event +'"  href="http://ensprod-dev-01.ebi.ac.uk/api/thomas/dc/download_datacheck_outputs/'+ row.id  +' ">'+ row.id +'</a>'
```

Comment:
> @vinay-ebi , @marcoooo I don't think the PR should have been approved/merged, because it does not fully work...

As you mentioned there are few hardcoded path which i created for demo and didnt remove . i will make the required changes to layout as per comments